support/objc-support.m

```diff
@@ -61,3 +61,29 @@ NSComparisonResult mono_embeddinator_compare_to (MonoEmbedObject *object, MonoMe
 	int gchandle = (int) [native performSelector:@selector (xamarinGetGCHandle)];
 	return mono_gchandle_get_target (gchandle);
 }
+
+NSDecimalNumber* mono_embeddinator_get_nsdecimalnumber (void* __unboxedresult)
+{
+	static NSDictionary* __forcedotseparator = nil;
```

Comment:
You don't need all the ugly underscores in the variable names when the code isn't generated :smile: